from nsj_rest_test_util.dao.exception.exception import ExcecaoLancadaManualmente


class RepetirException(ExcecaoLancadaManualmente):
    pass
